django/db/models/expressions.py

```diff
@@ -1287,17 +1287,6 @@ def __init__(self, *expressions, **extra):
         )
         super().__init__(*expressions, **extra)
 
-    def as_sql(self, *args, **kwargs):
-        if not self.source_expressions:
-            return "", ()
-        return super().as_sql(*args, **kwargs)
```

Comment:
This is not necessary anymore as `ExpressionList` requires at least one expression to be specified in `__init__`.

Reply:
Actually it's used by postgres' `OrderableAggMixin` so it will require some adjustments over there to avoid creating an `OrderByList` in the first place.

Reply:
Pushed a second commit to this effect.